mapstore2_adapter/plugins/geonode.py

```diff
@@ -373,7 +373,7 @@ def get_overlays(self, viewer, request=None):
                                 overlay['keywords'] = capa['keywords']
                             if 'dimensions' in capa and capa['dimensions']:
                                 overlay['dimensions'] = self.get_layer_dimensions(dimensions=capa['dimensions'])
-                            if 'storeType' in capa and 'vector' in capa['storeType']:
```

Comment:
The change to subtype has probably gone too far... These `capa` dicstionary should reflect the original OGC capabilites, doesn't it? Now it contains `subtype` instead of `storeType` (the original OGC element). 
By the way, if this `capa` dictionary is now indpendet from the concrete Resource Manager I'm fine with this. Opinions @afabiani?